errors/src/main/java/com/palantir/conjure/java/api/errors/AbstractServiceError.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * (c) Copyright 2025 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.api.errors;
+
+import com.palantir.logsafe.SafeLoggable;
+
+/**
+ * This class should not be used directly. The parent class for ServiceException and EndpointServiceException.
+ */
+public abstract class AbstractServiceError extends RuntimeException implements SafeLoggable {
```

Comment:
nit: Could we name this `AbstractServiceException` since it's not an `Error`?
```suggestion
public abstract class AbstractServiceException extends RuntimeException implements SafeLoggable {
```